plugins/cookie_manager/test/cookies_test.dart

```diff
@@ -49,6 +49,32 @@ void main() {
       HttpHeaders.cookieHeader: mockSecondRequestCookies,
     });
 
+    cookieManager.onRequest(options, mockRequestInterceptorHandler);
+  });
+  test('testing set-cookies parsing', () async {
+    const List<String> mockResponseCookies = [
+      "key=value; expires=Sun, 19 Feb 3000 00:42:14 GMT; path=/; HttpOnly; secure; SameSite=Lax",
+      "key1=value1; expires=Sun, 19 Feb 3000 01:43:15 GMT; path=/; HttpOnly; secure; SameSite=Lax, key2=value2; expires=Sat, 20 May 3000 00:43:15 GMT; path=/; HttpOnly; secure; SameSite=Lax",
```

Comment:
I don't see any single quotes inside, so it should be wrapped with single quotes.